package.json

```diff
@@ -74,7 +74,7 @@
     "terser": "^5.9.0",
     "unistore": "^3.4.1",
     "web-vitals": "^2.1.4",
-    "webdev-infra": "^1.0.28",
+    "webdev-infra": "^1.0.30",
```

Comment:
While https://github.com/GoogleChrome/webdev-infra/pull/21 was committed to GitHub, there hasn't been a corresponding publish of `v1.0.30` to `npm`: https://www.npmjs.com/package/webdev-infra

There are some instructions on publishing at https://github.com/GoogleChrome/webdev-infra#-important, but all of the previous versions have been published by folks who are no longer on the team.

@devnook, do you have publish rights? If not, I think we can pull in some folks with admin access to help.

Reply:
Published now.